android/sdl_android/src/main/java/com/smartdevicelink/managers/video/resolution/VideoStreamingRange.java

```diff
@@ -0,0 +1,64 @@
+package com.smartdevicelink.managers.video.resolution;
+
+public class VideoStreamingRange {
+    private Resolution minResolution;
+    private Resolution maxResolution;
+    private Double minScreenDiagonal;
+    private AspectRatio aspectRatio;
+
+    public VideoStreamingRange(
+            Resolution minResolution,
+            Resolution maxResolution,
+            Double minScreenDiagonal,
+            AspectRatio aspectRatio
+    ) {
```

Comment:
This does not match the proposal and should replace `AspectRatio aspectRatio` with `Double minAspectRatio` and `Double maxAspectRatio`

From the proposal:
public class VideoStreamingRange {
    private Resolution minResolution;
    private Resolution maxResolution;
    private Double minAspectRatio;
    private Double maxAspectRatio;
    private Double minScreenDiagonal;
}

Reply:
@RHenigan Thanks for your reply, these were the code examples, and to make the IOS/Android libraries mirroring each other. The @santhanamk, @akaremo have approved these changes as well previously, let me know if we still need to move forward strictly with the proposal

Reply:
@kostyaBoss This does not appear to mirror the iOS implementation, the iOS implementation has these 5 properties:
```
@property (nonatomic, assign) float minimumAspectRatio;

// The maximum supported normalized aspect ratio, Min value is 1
@property (nonatomic, assign) float maximumAspectRatio;

// The minimum supported diagonal screen size in inches, defaults to 0 (0 matches any size)
@property (nonatomic, assign) float minimumDiagonal;

// The minimum resolution to support, it overrides .minimumAspectRatio
@property (nonatomic, strong, nullable) SDLImageResolution *minimumResolution;

// The maximum resolution to support, it overrides .maximumAspectRatio
@property (nonatomic, strong, nullable) SDLImageResolution *maximumResolution;
```

This class should be updated to match the proposal and the iOS implementation